CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## Unreleased
 
+FEATURES:
+* `r/tfe_registry_module`: Add `source_directory` and `tag_prefix` registry module support for private registry monorepository, by @jillirami ([#1800](https://github.com/hashicorp/terraform-provider-tfe/pull/1800))
```

Comment:
Also mention that it's beta here!
```suggestion
* `r/tfe_registry_module`: Add `source_directory` and `tag_prefix` registry module support for private registry monorepository, which is a beta feature and not available to all users, by @jillirami ([#1800](https://github.com/hashicorp/terraform-provider-tfe/pull/1800))
```

Reply:
Updated, thank you for pointing this out!